job-dsl-plugin/src/main/groovy/javaposse/jobdsl/plugin/casc/SeedJobConfigurator.java

```diff
@@ -23,6 +25,7 @@
 import java.util.Collections;
 import java.util.List;
 import java.util.Set;
+import java.util.stream.Stream;
```

Comment:
The import order still looks funky but I guess that is the job for the maintainer to fix 😅 
In the repos, I maintain I prefer to keep A-Z and static last, no star import at all even in tests.

Reply:
Yes, I am going to fix that.

Reply:
@daspilker you can have a look at https://github.com/casz/configuration-as-code-plugin we have added IDEA codestyle and checkstyle into maven.